docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -5,4 +5,26 @@ This guide contains instructions to upgrade from version v7.1.0 to Unreleased.
 **Before you start, don't forget to take a look at [general instructions](/UPGRADE.md) about upgrading.**
 There you can find links to upgrade notes for other versions too.
 
+## [shopsys/project-base]
+### Configuration
+- translations are now extracted also from overwritten templates in `app/Resources` ([#931](https://github.com/shopsys/shopsys/pull/931))
```

Comment:
```suggestion
```

Reply:
Hello @boris-brtan,

I have updated my UPGRADE instructions in little different way, but it should be as expected now. I have also rebased on current master.